### ISSUE TITLE
render throws when passing an empty string as children

### ISSUE THREAD

**Issue by rvdkooy** [martinandert/react-interpolate-component#2]
Hi,

When I use React-translate-component and the resource value is an empty string
The underlying interpolate component throws a `Interpolate expects either a format string as only child or a 'format' prop with a string value` error

this is because 

```
this.props.children || this.props.format --> results in undefined
example:
"" || undefined => undefined
```

see: https://github.com/martinandert/react-interpolate-component/blob/master/index.js#L22

Is this expected behavior?

with kind regards,

Ronald van der Kooij

**Comment by martinandert** [comment]
Hi Ronald, this has been fixed with v0.8.1.

Thanks for reporting it!

**Comment by rvdkooy** [comment]
Thanks a lot!
